### PR TITLE
style(lemon-ui): Make `Sparkline` look great

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

This sparkline (well, sparkbar) kind of worked but didn't look great:

Screenshot 2024-01-17 at 16 58 06

Very hard to say where the graph ends and where it starts, where's 0 or what order of magnitude we're looking at. If there were zero values, especially on the edges, you could think that's a layout bug. And then there was some visual roughness, particularly in large spaces between bars.

## Changes

Gave `Sparkline` some love so that it's more useful. These improvements are also shared by the existing places where `Sparkline` is used, so the improvement is app-wide.

Screenshot 2024-01-17 at 16 56 32

### CODE REVIEW

**Comment by Twixes** [comment]
They're one and the same component, so HogQL sparklines will also look better @mariusandra

File: ee/tasks/subscriptions/__init__.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 from ee.tasks.subscriptions.email_subscriptions import send_email_subscription_report
 from ee.tasks.subscriptions.slack_subscriptions import send_slack_subscription_report
 from ee.tasks.subscriptions.subscription_utils import generate_assets
-from posthog import settings
```
Comment (thmsobrmlr) [inline ee/tasks/subscriptions/__init__.py:12]: Is this intentionally removed here?

**Review by thmsobrmlr — APPROVED** [review]
Good to have someone who cares about the usability & looks of things :)

A couple of things feel a bit off to me, but I'm going to let you decide what you think makes sense:
- The previous (yellow) highlight on hover was more visible. The current one is almost not noticable.
- I wouldn't always display the label to the left, as the sparkline is just meant to give a rough indication of proportions or a rate change.
- I personally don't like the rounded borders on top and would stick to square bars.
- A little more space (just 1-2 pixels) between the bars could work well.

File: ee/tasks/subscriptions/__init__.py (hunk 1 above)
Comment (Twixes) [reply ee/tasks/subscriptions/__init__.py:12]: Yup, looks like this was merged to `master` without passing `ruff` checks and now I weren't able to merge this branch with master without resolving the duplication :/

**Comment by Twixes** [comment]
Thanks for your thoughts @thmsobrmlr, useful stuff!

I agree with you on bar spacing and rounding, tuned that:

Screenshot 2024-01-17 at 19 04 47

I also added an option to disable the maximum indicator with `maximumIndicator={false}`. I'll leave it on by default though, as it feels pretty useful in most of our use cases (I got the idea from Sentry's UI).

The subtle hover effect is intentional, as previously it felt very strong for a non-clickable element. It was also cumbersome to extend to custom color series, which is the direct reason for going lighter.

**Pull request merged**